Formula/libngspice.rb

```diff
@@ -37,6 +28,14 @@ class Libngspice < Formula
     depends_on "libtool" => :build
   end
 
+  conflicts_with "ngspice", because: "both install initdata scripts"
+
+  # Fix -flat_namespace being used on Big Sur and later.
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/03cf8088210822aa2c1ab544ed58ea04c897d9c4/libtool/configure-big_sur.diff"
+    sha256 "35acd6aebc19843f1a2b3a63e880baceb0f5278ab1ace661e57a502d9d78c93c"
+  end
```

Comment:
Let's keep this in the `stable` block since we don't want to apply this patch to `head`.

Reply:
👍